### DISCUSSION TITLE
Shard failure

### DISCUSSION THREAD

**Discussion by jdecker79** [Security-Onion-Solutions/securityonion#3736] [Unsupported Versions]
When using Kibana I receive a message that 1 of 202 shards failed.  I have tried to research the error but am not having any luck finding anything that moves me in a positive direction.  Any help on what I need to look at to fix this error would be greatly appreciated.

This is the failure message:
illegal_argument_exception at shard 0index onion-manager:so-ossecnode IE-luOShQHSmSP7punWNbw
Type
illegal_argument_exception
Reason
Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [rule.uuid] in order to load field data by uninverting the inverted index. Note that this can use significant memory.

**Comment by weslambert** [comment]
Are you still having this issue?

**Reply by jdecker79** [reply]
Yes.  I am very new to security onion and learning as I go, so was hoping to get some direction on solving the issue.

**Reply by jdecker79** [reply]
I still am experiencing this error, and have not been able to find any information to help me resolve the error.

**Reply by weslambert** [reply]
In Kibana, if you go to Stack Management -> Index Patterns -> `*:so-*` and search for rule.uuid, what data type is shown for the field mapping?

**Reply by jdecker79** [reply]
It shows Type as string, Format Url, with blue dots under Searchable and Aggregatable, and nothing under Excluded.
